core/services/versionchooser/utils/chooser.py

```diff
@@ -128,12 +125,12 @@ async def set_version(self, image: str, tag: str) -> web.StreamResponse:
                 startup_file.truncate()
 
                 logging.info("Starting bootstrap...")
-                bootstrap = self.client.containers.get("companion-bootstrap")
-                bootstrap.start()
+                bootstrap = await self.client.containers.get("companion-bootstrap")  # type: ignore
+                await bootstrap.start()
 
                 logging.info("Stopping core...")
-                core = self.client.containers.get("companion-core")
-                core.kill()
+                core = await self.client.containers.get("companion-core")  # type: ignore
+                await core.kill()
```

Comment:
Just to understand, what is the type problem here?

Reply:
IIRC it is just that there are no stubs for aiodocker, so mypy was complaining about no type information